net-misc/tigervnc/tigervnc-1.10.0.ebuild

```diff
@@ -0,0 +1,197 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools cmake-utils eutils flag-o-matic java-pkg-opt-2 systemd xdg-utils
+
+XSERVER_VERSION="1.20.6"
+
+DESCRIPTION="Remote desktop viewer display system"
+HOMEPAGE="https://tigervnc.org https://github.com/TigerVNC/tigervnc"
+SRC_URI="https://github.com/TigerVNC/tigervnc/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	server? ( ftp://ftp.freedesktop.org/pub/xorg/individual/xserver/xorg-server-${XSERVER_VERSION}.tar.bz2	)"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86"
+IUSE="dri3 +drm gnutls java libressl nls +opengl pam server xinerama +xorgmodule"
+
+COMMON_DEPEND="
+	>=x11-libs/fltk-1.3.1
+	sys-libs/zlib:=
+	virtual/jpeg:0
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXrender
+	gnutls? ( net-libs/gnutls:= )
+	nls? ( virtual/libiconv )
+	pam? ( sys-libs/pam )
+	xinerama? ( x11-libs/libXinerama )
+	server? (
+		x11-apps/xsetroot
+		x11-libs/libXau
+		x11-libs/libXdamage
+		x11-libs/libXdmcp
+		x11-libs/libXfont2
+		x11-libs/libXtst
+		>=x11-apps/xauth-1.0.3
+		>=x11-libs/pixman-0.27.2
+		>=x11-misc/xkeyboard-config-2.4.1-r3
+		drm? ( x11-libs/libdrm )
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+		xorgmodule? ( =x11-base/xorg-server-${XSERVER_VERSION%.*}* )
+	)
+	"
+
+RDEPEND="${COMMON_DEPEND}
+	!net-misc/tightvnc
+	!net-misc/vnc
+	!net-misc/xf4vnc
+	java? ( >=virtual/jre-1.5:* )"
+
+DEPEND="${COMMON_DEPEND}
+	media-libs/fontconfig
+	x11-base/xorg-proto
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libXcursor
+	x11-libs/libXfixes
+	x11-libs/libXft
+	x11-libs/libXi
+	java? ( >=virtual/jdk-1.5 )
+	nls? ( sys-devel/gettext )
+	server? (
+		>=x11-libs/xtrans-1.3.3
```

Comment:
Version is always satisfied, and this is out of place in an alphabetical ordering.